generic-templates/newsletter-signup-snippet/snippet.html

```diff
@@ -435,10 +435,18 @@
         var trackItemWidth = 420;
         var currentIndex = 0;
 
-        emailInput.addEventListener("focus", function() {
+        emailInput.addEventListener("click", function() {
           emailInput.classList.remove("invalid");
         });
-        function onNext(e) {
+
+        document.addEventListener("keydown", function(e) {
```

Comment:
I didn't think of that, but divs don't send keydown events by default because they are not focusable by default. The document is focusable.

I can put tabindex on it to make it selectable, thus focusable, thus, fires keydown events, but feels really hacky to me.

If the search bar is focused, `onNext` shouldn't be called. So far it's the cleanest solution I could come up with, but I'm not particularly fond of it.